terraform/deployments/ecr/main.tf

```diff
@@ -21,32 +25,28 @@ provider "aws" {
   }
 }
 
-# TODO: Get rid of this list and just give CI permission to create repos, e.g.
-# with https://github.com/byu-oit/github-action-create-ecr-repo-if-missing
+# NOTE: Uses GITHUB_TOKEN env var, an OAuth / Personal Access Token, for auth
+provider "github" {
+  owner = "alphagov"
+}
+
+data "github_repositories" "govuk" {
+  query = "org:alphagov topic:container topic:govuk fork:false archived:false"
```

Comment:
Thanks for tagging these up with `topic:container`. I think that'll avoid creating a bunch of unused/empty container repos in the registry (which would degrade the UI/UX) so this is a win for usability.